### ISSUE TITLE
Lack of accessibility for the blind

### ISSUE THREAD

**Issue by tmasterson** [elementor/elementor#13792]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

I have a site that was originally designed using elementor by a sighted person.  I now have to maintain that site but as elmentor is not accessible with a screen reader and just using the keyboard (no mouse) I have to try and do edits in either the block or classic editor (preffered).  However as elmentor appears to keep a copy of the pages somewhere else any changes I make using these other editors does not take effect.  I need a way to either have these changes translate to elmentor, have elementor made accessible (doesn'at appear to be even a remote priority), or easily convert the entire site away from elementor.

Note I did not put this on facebook as I use that as little as possible and will not singn up for a group I do not want to be a part of.
## Steps to reproduce

Everything is stated in the description.  I understand that accessibility is often overlooked in design but it should not be as it is much harder to put it in later.
## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [ ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by DorShahar** [comment]
Hi @tmasterson 
Thanks for reaching out.
We aware of the accessibility problems that we have with the product, and we are trying to improve it.
For example - in v3.1.0 we introduced the "Accessibility Improvements" experiment (you can reach it under WP dashboard > Elementor > Settings > Experiments). 

We handle our Tabs widget, and next on the list is the Accordion widget.

I'm closing the issue, but please keep raising up specific A11y problems related to widgets or features. 

Thanks
